testing/integration/test/bin/load-zork.sh

```diff
@@ -47,14 +51,20 @@ if $RUNVNC; then
     x11vnc -display :10 -forever &
 fi
 
-if ! $PREBUILT; then
+if [[! $PREBUILT ]] || $NPM; then
     mkdir -p /test/src
     cd /test/src
     npm install -g bower grunt-cli
```

Comment:
No need to install these globally if we're using prebuilt NPMs. I think you want to move this - and the `grunt zork` command at line 68 - into the `if` block starting at line 63.